src/core/Track.cpp

```diff
@@ -52,25 +50,29 @@
  *
  * \todo check the definitions of all the properties - are they OK?
  */
-Track::Track( TrackTypes type, TrackContainer * tc ) :
-	Model( tc ),                   /*!< The track Model */
-	m_trackContainer( tc ),        /*!< The track container object */
-	m_type( type ),                /*!< The track type */
-	m_name(),                       /*!< The track's name */
-	m_mutedModel( false, this, tr( "Mute" ) ), /*!< For controlling track muting */
-	m_soloModel( false, this, tr( "Solo" ) ), /*!< For controlling track soloing */
-	m_simpleSerializingMode( false ),
-	m_trackContentObjects(),        /*!< The track content objects (segments) */
-	m_color( 0, 0, 0 ),
-	m_hasColor( false )
+Track::Track(TrackTypes type, TrackContainer* tc)
```

Comment:
This looks kind of broken/awkward now.

Reply:
Same here, it seems to take the spacing and comments from the header and inserts them here. I'll ask on StackOverflow.

Reply:
Oh, I just see this was originally wrong already. Doxygen trailing comments should always go to the declaration, not into the CTOR initializer list. So IMO we should fix the comments before running clang-format here.

Reply:
Resolving, since this is a code issue IMO.